pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -275,11 +275,15 @@ type Controller struct {
 	Driver *ControllerDriver `xml:"driver,omitempty"`
 }
 
+// END Controller -----------------------------
+
+// BEGIN ControllerDriver
 type ControllerDriver struct {
 	IOThread *uint `xml:"iothread,attr,omitempty"`
+	Queues   *uint `xml:"queues,attr,omitempty"`
```

Comment:
It seems like the Queues added here to `ControllerDriver` is not really used. Can it be that you only use `Queues` in `DiskDriver`?